src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1405,15 +1405,6 @@ void ApplicationManagerImpl::SendMessageToMobile(
       return;
     }
 
```

Comment:
The style checker also has an issue with the extra newline which separates the two end braces, I should be able to fix this quickly through GitHub